persistence-model/src/main/scala/hmda/persistence/messages/events/processing/HmdaFileParserEvents.scala

```diff
@@ -0,0 +1,14 @@
+package hmda.persistence.messages.events.processing
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.model.parser.LarParsingError
+import hmda.persistence.messages.CommonMessages.Event
+
+object HmdaFileParserEvents {
+  trait HmdaFileParserEvent extends Event
```

Comment:
Better type safety. They all inherit from `Event`, which is too generic. One use case might be when reading the journal, maybe I only care about a certain "family" of events.